chapter03/mdw/Ex-1.pl

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/perl
+
+use 5.010;
+use warnings;
+
+say "Enter Strings to be reversed:";
+
+chomp(@strings=<>);
+
+@result = reverse @strings;
+
+foreach $result (@result) {
+  say $result;
+};
```

Comment:
You could just avoid using chomp and then print the array. It saves removing the new lines and then adding them again.